src/Test/EntityManagerMockFactoryTrait.php

```diff
@@ -21,6 +21,11 @@
 use Doctrine\Persistence\Mapping\ClassMetadata;
 use PHPUnit\Framework\MockObject\MockObject;
 
+/**
+ * NEXT_MAJOR: Remove this class.
```

Comment:
IMO We should deprecate this trait, is makes no sense to expose as a trait a code that mocks the entity manager. wdyt @VincentLanglet ?

I only saw it used on ClassificationBundle, and we can easily adapt that code.

Reply:
Sure, it's used in https://github.com/sonata-project/SonataClassificationBundle/blob/ae001e0928c83236343319e1610512bad73f43a2/tests/Entity/TagManagerTest.php

Can you provide a PR to stop using it ?